### PR TITLE
Added default Cornell Box scene and benchmark application

### PR DESCRIPTION
**Pull request by favreau**
Added default Cornell Box scene when no datasource is provided. A new benchmarking application was also added to allow performance measurements.

### CODE REVIEW

**Comment by favreau** [comment]
retest this please

File: brayns/Brayns.cpp [hunk 1]
```diff
@@ -67,13 +67,17 @@ struct Brayns::Impl
 
         // Default sun light
         DirectionalLightPtr sunLight( new DirectionalLight(
-            Vector3f( 0.f, 0.f, 1.f ), Vector3f( 1.f, 1.f, 1.f ), 1.f ));
```
Comment (bilgili) [inline brayns/Brayns.cpp]: can yo do those - vecs -  in namespace{} as constants ?

File: brayns/common/scene/Scene.cpp [hunk 2]
```diff
@@ -154,6 +155,110 @@ MaterialPtr Scene::getMaterial( size_t index )
...
+        { -0.3f, 0.98f, -0.3f },
+        {  0.3f, 0.98f, -0.3f },
+        {  0.3f, 0.98f,  0.3f },
+        { -0.3f, 0.98f,  0.3f }
+    };
+    for( size_t i = 0; i < 4; ++i )
+        _trianglesMeshes[material].getVertices().push_back( lampPositions[i] );
+    _trianglesMeshes[material].getIndices().push_back( Vector3i( 0, 1, 2 ));
+    _trianglesMeshes[material].getIndices().push_back( Vector3i( 2, 3, 0 ));
+    _materials[material]->setColor( Vector3f( 1.f, 1.f, 1.f ));
```
Comment (bilgili) [inline brayns/common/scene/Scene.cpp]: color can be constant, as before.

Comment (favreau) [inline brayns/common/scene/Scene.cpp]: True! Thanks

**Pull request merged**